src/common/components/ui/OpenShiftVersionDropown.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  DropdownItem,
+  DropdownToggle,
+  Dropdown,
+  DropdownProps,
+  HelperText,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+
+import { OpenshiftVersionOptionType } from '../../types';
+import { TFunction } from 'i18next';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+
+export type HelperTextType = (
+  versions: OpenshiftVersionOptionType[],
+  value: string,
+  t: TFunction,
+) => JSX.Element | null;
+
+type OpenShiftVersionDropdownProps = {
```

Comment:
I think we can clean up the properties a bit and make the component simpler.

- `idPrefix` and  `position` are not used.
-`helperTextDefault` is unnecessary. It can be handled from `OpenShiftVersionDropdown` by calling `getHelperText` with `defaultValue`.
- `defaultValue` can be defined in `OpenShiftVersionDropdown` as it's just the label of the first item.